docs/setuptools.rst

```diff
@@ -45,12 +45,16 @@ Python package and a ``setup.py`` file.
 
 Imagine this directory structure::
 
+.. code-block:: console
+
     yourscript.py
     setup.py
 
 Contents of ``yourscript.py``:
 
-.. click:example::
+.. click:example:
```

Comment:
Don't do this, it breaks Click's Sphinx integration, and it's already highlighted correctly.